scan-results.js

```diff
@@ -6,28 +6,28 @@ const HIBP = require("./hibp");
 const sha1 = require("./sha1-utils");
 
 // Growth Experiment
-// const { getExperimentBranch } = require("./controllers/utils");
-// const AppConstants = require("./app-constants");
-// const EXPERIMENTS_ENABLED = (AppConstants.EXPERIMENT_ACTIVE === "1");
+const AppConstants = require("./app-constants");
+const EXPERIMENTS_ENABLED = (AppConstants.EXPERIMENT_ACTIVE === "1");
 
 const scanResult = async(req, selfScan=false) => {
 
   const allBreaches = req.app.locals.breaches;
   let scannedEmail = null;
 
   // Growth Experiment
-  // let experimentBranch = null;
-  // let isUserInExperiment = null;
-  // let experimentBranchB = null;
+  let experimentBranch = null;
+  let isUserInExperiment = null;
+  let experimentBranchB = null;
```

Comment:
`let` = code sniff. Can we wrap these up in a single object and do something more like:
```js
const experimentFlags = _getExperimentFlags(req);
...
res.render("dashboards", {
...
  experimentFlags,
});
```